requirements/app/base.txt

```diff
@@ -9,3 +9,4 @@ traitlets>=5.3.0, <=5.4.0
 arrow>=1.2.0, <1.2.4
 lightning-utilities==0.3.*
 beautifulsoup4<=4.8.2
+psutil
```

Comment:
Might want to set a min/max version as in all other dependencies

Reply:
Done !